source/reference/release-notes.txt

```diff
@@ -42,6 +42,31 @@ The {+driver-short+} 2.22 release includes the following new features:
 
 - Adds compatibility with {+language+} 3.3.
 
+BSON v5.1
+~~~~~~~~~
+
+The ``bson`` gem is the implementation of the BSON specification in
+{+language+}. ``bson`` v5.1 accompanies the {+driver-short+} v2.22
+release.
```

Comment:
We actually discovered that all the changes we meant to release in driver v2.22 somehow got released in 2.21.1. This includes the vector search support in the driver, though it wouldn't have worked until bson 5.1.0.

So, bson 5.1.0 supports ruby driver 2.21.1. There will be a driver 2.22 eventually, but probably not imminently.